webapp/channels/src/components/sidebar/sidebar_list/sidebar_list.tsx

```diff
@@ -133,6 +136,7 @@ export default class SidebarList extends React.PureComponent<Props, State> {
         this.state = {
             showTopUnread: false,
             showBottomUnread: false,
+            autoHide: false,
```

Comment:
```suggestion
            autoHide: true,
```

Reply:
I think we want the initial state set to false.

Reply:
**updated** Good catch, I changed it to default `true`